cesium_app/tests/test_util.py

```diff
@@ -37,24 +37,64 @@ def test_check_model_param_types():
                 "class_weight": {'a': 0.2, 'b': 0.8}}
     util.check_model_param_types(model_type, params)
 
+    params = {"n_estimators": 100.1}
+    try:
+        util.check_model_param_types(model_type, params)
+    except ValueError:
+        pass
+    else:
+        raise Exception("check_model_param_types did not error when it should have")
```

Comment:
This could all be one line w/ something like `pytest.raises`: http://doc.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions